app/controllers/review_mapping_controller.rb

```diff
@@ -132,7 +132,8 @@ def assign_quiz_dynamically
         @map = QuizResponseMap.new
         @map.reviewee_id = Questionnaire.find(params[:questionnaire_id]).instructor_id
         @map.reviewer_id = params[:participant_id]
-        @map.reviewed_object_id = Questionnaire.find_by_instructor_id(@map.reviewee_id).id
+        #@map.reviewed_object_id = Questionnaire.find_by_instructor_id(@map.reviewee_id).id
```

Comment:
Please delete original code, instead of comment it out. Git will record your modification.